src/app/public/src/modules/wrapper/wrapper.component.ts

```diff
@@ -94,6 +100,11 @@ export class StacheWrapperComponent implements OnInit, AfterContentInit, OnDestr
     );
   }
 
+  private checkEditButtonUrl(): boolean {
+    const url = _get(this.config, 'skyux.appSettings.stache.editButton.url');
+    return (url !== undefined) ? true : false;
```

Comment:
`(url !== undefined)` already computes to a boolean.

Reply:
👍